bin/git-effort

```diff
@@ -207,11 +207,15 @@ heading
 
 # send paths to effort
 nPaths=${#paths[@]}
+nProcs=$(getconf NPROCESSORS_ONLN)
```

Comment:
I get `getconf: Unrecognized variable `NPROCESSORS_ONLN'` when running this command. We should provide a fallback for it.

Reply:
Sure, since I can not find a perfect command to support all distributions, platforms, I simply choose 8 as a magic number.